internal/util/filesystem.go

```diff
@@ -29,6 +29,8 @@ const SchemasDirectory = "_schemas"
 // TestDataDirectory is the name of the special directory containing test fixtures. It is defined here to avoid an import loop.
 const TestDataDirectory = "testdata"
 
+const pathSeparator = string(filepath.Separator)
```

Comment:
This is also defined in `cmd/cerbosctl/hub/store/store.go`, _could_ make it public and share the same type 🤷‍♂️